server/routes/imgUpload.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express');
+const router = express.Router();
+const path = require('path');
+const multer = require('multer');
+const { uploadImages } = require('../controllers/imgUpload');
+
+const fileStorageEngine = multer.diskStorage({
```

Comment:
We can add a file filter too using Multer

Ref: https://stackoverflow.com/questions/38652848/filter-files-on-the-basis-of-extension-using-multer-in-express-js